### PR TITLE
Rename Obsolete to Archived in UI and tests

### PR DESCRIPTION
**Pull request by djansen-redhat**

### CODE REVIEW

File: functional-test/src/main/java/org/zanata/page/groups/VersionGroupsPage.java [hunk 1]
```diff
@@ -68,8 +68,8 @@ public VersionGroupPage goToGroup(String groupName) {
         return new VersionGroupPage(getDriver());
     }
 
-    public VersionGroupsPage toggleObsolete(final boolean show) {
-        WebElement showObsolete = waitForWebElement(toggleObsolete);
+    public VersionGroupsPage toggleArchived(final boolean show) {
+        WebElement showObsolete = waitForWebElement(toggleArchived);
         if (show != showObsolete.isSelected()) {
             showObsolete.click();
```
Comment (davidmason) [inline functional-test/src/main/java/org/zanata/page/groups/VersionGroupsPage.java]: This could be `showArchived` too.

**Comment by davidmason** [comment]
:+1: looks good, has one more easy place to replace. It would be good to get all the keys in `messages.properties` updated too, but we don't have decent refactor support for that so I wouldn't block this change on it.

**Comment by djansen-redhat** [comment]
@davidmason thanks, updated

**Comment by davidmason** [comment]
:+1:

**Pull request merged**